themes/classic/templates/customer/_partials/block-address.tpl

```diff
@@ -27,6 +27,8 @@
     <div class="address-body">
       <h4>{$address.alias}</h4>
       <address>{$address.formatted nofilter}</address>
+      {* To display the fields value extra added in a address from by using hook 'additionalCustomerAddressFields' *}
+      {hook h='displayAdditionalCustomerAddressFields' address=$address}
```

Comment:
Is this modification directly linked to this issue? #9663

Reply:
@matthieu-rolland 
No, this modification is not linked to this issue.

Reply:
Could you please remove this? You can put it in another dedicated pull request if you think this feature is important 👍